tools/src/blockchain-analyser/Rendering.hs

```diff
@@ -0,0 +1,197 @@
+
+module Rendering ( render
+                 , renderBlock
+                 , renderBlocks
+                 , renderHeader
+                 ) where
```

Comment:
The way we format export lists is defined [here](https://github.com/input-output-hk/cardano-sl-style-guides/blob/master/haskell-style-guide.md#export-lists).
I know it's minor, but I want it to be consistent with other code. Note that you module `Options` follows that rule, so even in this PR formatting is different. When it's different, some people will change it, it will lead to unnecessary extra diff and noise, sometimes even conflicts.